jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -59,46 +78,116 @@
 	@Resource
 	protected javax.ejb.TimerService timerService;
 
+    @Resource
+    protected javax.ejb.SessionContext sessionContext;
```

Comment:
maybe directly inject UserTransaction instead of session context...

Reply:
done